app/models/ability.rb

```diff
@@ -3,30 +3,56 @@ class Ability
 
   # Initializes the ability class
   def initialize(user)
-    # Order Abilities
-    # (Check https://github.com/CanCanCommunity/cancancan/wiki/Ability-Precedence)
-    # Check roles of user, using rolify. Role name is *case sensitive*
-    # user.is_organizer? or user.has_role? :organizer
-    # user.is_cfp_of? Conference or user.has_role? :cfp, Conference
-    # user.is_info_desk_of? Conference
-    # user.is_volunteers_coordinator_of? Conference
-    # user.is_attendee_of? Conference
-    # The following is wrong because a user will only have 'cfp' role for a specific conference
-    # user.is_cfp? # This is always false
-
     user ||= User.new
 
-    # This is what sets up the different abilities
     if user.new_record?
       not_signed_in
-    # Checks if the user does not have any role and is not an admin
-    elsif user.roles.any? || user.is_admin
```

Comment:
Nothing special! I just thought it better to write it in a single line rather than checking each role separately.